scripts/install.sh

```diff
@@ -206,6 +216,21 @@ while [ "$1" != "" ]; do
         -r | --region )                 shift
                                         region=$1
                                         ;;
+        --lambdaLatencyThreshold )         shift
+                                        lambdaLatencyThreshold=$1
+                                        ;;
+        --apigatewayServerErrorThreshold )      shift
+                                        apigatewayServerErrorThreshold=$1
+                                        ;;
+        --apigatewayBadRequestErrorThreshold )  shift
+                                        apigatewayBadRequestErrorThreshold=$1
+                                        ;;
+        --lambdaErrorThreshold )                  shift
+                                        lambdaErrorThreshold=$1
+                                        ;;
+        --ddbToESLambdaErrorThreshold )                 shift
```

Comment:
Can you fix the indendation of the 'shift' word on all these options?

Reply:
sure.